### ISSUE TITLE
Support for Spacy 3.4.0

### ISSUE THREAD

**Issue by kodymoodley** [Pandora-Intelligence/crosslingual-coreference#22]
Hi, I would like to use this nice package for Dutch language models that only work with Spacy 3.4.0+. How difficult would it be to support spacy 3.4.0?

**Comment by stdweird** [comment]
@davidberenstein1957  is there a reason why the pyprojct toml locks the spacy version to 3.1.X?  (the readme even mentions using spacy 3.3+)

**Comment by davidberenstein1957** [comment]
Hi @stdweird @kodymoodley, I've not been too involved with maintaining this package. I will block some time to fix this coming weekend, but feel free to create a PR on top of this yourself too. I've also been wanting to fine-tune a coref model using another new package but have not gotten around to doing that as yet.

**Comment by stdweird** [comment]
@davidberenstein1957 do you suspect that we need to lock spacy (eg `> 3.0, < 3.8` or simpy allow all 3.X).
and if we can run the tests, it should be enough?

**Comment by davidberenstein1957** [comment]
Yes it should be.

**Comment by matesaki** [comment]
@davidberenstein1957 
Problem is with project [allennlp](https://github.com/allenai/allennlp) which was ended on the [version 2.10.1](https://pypi.org/project/allennlp/#history) with depends on [spacy](https://pypi.org/project/spacy/#history) (>=2.1.0,<3.4)

poetry lock:
SolverProblemError
Because crosslingual-coreference depends on allennlp (==2.10.1) which depends on spacy (>=2.1.0,<3.4), spacy is required.

**Comment by davidberenstein1957** [comment]
Ahh shit. I knew that in my subconscience. I will fine-tune a new model and wrap out allennlp with fast-coref :)

**Comment by hardchor** [comment]
Great to hear this project is still alive! Were you able to make any progress on this?

**Comment by davidberenstein1957** [comment]
Yes I was but I'm currently on holidays. I've prepped the data and will run some training later this month. 🙂El 9 may 2024, a las 2:43, Burkhard Reffeling ***@***.***> escribió:﻿
Great to hear this project is still alive! Were you able to make any progress on this?
